src/modules/getMeals.js

```diff
@@ -1,36 +1,85 @@
+/* eslint-disable radix */
+/* eslint-disable consistent-return */
 /* eslint-disable no-await-in-loop */
-const ids = [
-  52891,
-  53039,
-  53047,
-  52833,
-  52953,
-  53009,
-];
-
+const InvolvementApiKey = 'H1778Eipl0PRUFSJ8DNo';
 const content = document.getElementById('content');
-
 const ul = document.createElement('ul');
+
 ul.classList.add('meals-list');
+ul.id = 'classlist';
 content.appendChild(ul);
-const getMeals = async () => {
-  for (let i = 0; i < ids.length; i += 1) {
-    const response = await fetch(`https://www.themealdb.com/api/json/v1/1/lookup.php?i=${ids[i]}`);
+
+async function postLike(meal) {
+  const url = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${InvolvementApiKey}/likes/`;
+
+  const data = {
+    item_id: meal,
+  };
+
+  const param = {
+
+    method: 'Post',
+
+    headers: {
+
+      'content-type': 'application/json;charset = UTF-8',
+
+    },
+
+    body: JSON.stringify(data),
+
+  };
+
+  const sendRequest = await fetch(url, param);
+  const response = await sendRequest.text();
+  return response;
+}
+
+async function getLikes(itemId) {
+  const response = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${InvolvementApiKey}/likes`);
+  const data = await response.json();
+  for (let i = 0; i < data.length; i += 1) {
+    if (data[i].item_id === itemId) {
+      return data[i].likes;
+    }
+  }
+  // return data;
+}
+
+const getMeals = async (category) => {
+  for (let i = 0; i < 6; i += 1) {
+    const response = await fetch(`https://www.themealdb.com/api/json/v1/1/filter.php?c=${category}`);
     const data = await response.json();
```

Comment:
Nice implementation. Kindly consider extracting fetch function into a separate modules file. This will make your code more readable and reusable.